pkg/plugin/v2/scaffolds/internal/templates/dockerfile.go

```diff
@@ -39,7 +39,7 @@ func (f *Dockerfile) SetTemplateDefaults() error {
 }
 
 const dockerfileTemplate = `# Build the manager binary
-FROM golang:1.13 as builder
```

Comment:
We shouldn't be changing v2's Go version unless we account for plugin version in `checkGoVersion()`.

Reply:
Reverted v2's Go version to 1.13.